error[mismatched_arg_count]: this function takes 3 arguments but 4 arguments were supplied
  ┌─ reassigned_variables.lua:1:16
  │
1 │ local function foo(a, b, c)
  │                --- note: function defined here
  ·
5 │ foo(1, 2, 3, 4)
  │ ^^^ expected 3 arguments

error[mismatched_arg_count]: this function takes 3 arguments but 4 arguments were supplied
   ┌─ reassigned_variables.lua:1:16
   │
 1 │ local function foo(a, b, c)
   │                --- note: function defined here
   ·
10 │ foo(1, 2, 3, 4)
   │ ^^^ expected 3 arguments

error[mismatched_arg_count]: this function takes 3 arguments but 4 arguments were supplied
   ┌─ reassigned_variables.lua:1:16
   │
 1 │ local function foo(a, b, c)
   │                --- note: function defined here
   ·
20 │ foo(1, 2, 3, 4)
   │ ^^^ expected 3 arguments

